build-aux/flatpak/org.qelectrotech.QElectroTech.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "org.qelectrotech.QElectroTech",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.14",
+  "sdk": "org.kde.Sdk",
+  "command": "qelectrotech",
+  "rename-desktop-file": "qelectrotech.desktop",
+  "rename-appdata-file": "qelectrotech",
+  "rename-icon": "qelectrotech",
```

Comment:
This renames the desktop file, the appdata file, and the main launcher icons.

This is required because Flatpak enforces the aforementioned naming convention, and will only export out of the sandbox the files named like the application id.

Those files need to be exported outside of the sandbox, or else the desktop launcher (e.g an application menu) won't be able to see the app, and users will be very confused how to run it. :stuck_out_tongue: 

Similarly the appdata must be renamed for software centers like GNOME Software or KDE Discover to see it and offer users to install the app.

Renaming all of those directly in the source tree would be better and simpler, but it is not required to make a Flatpak build. I'm happy to send the changes in a future merge request if they are desired.

Reply:
We should probably put the renaming etc. of resource files into the qmake project iself. Similar to this: 

https://github.com/solvespace/solvespace/blob/master/res/CMakeLists.txt#L129

Reply:
I'd just plain remove the source files. But like I said that's not necessary for this merge request, so I'd rather do that separately.

My point here was to explain to the maintainers who might not know about the convention **what** this does and **why** it exists, not to start a conversation on **how** to do it in the future. (I've done it for multiple apps in the past)